### ISSUE TITLE
Income Segregation at a macro-level

### ISSUE THREAD

**Issue by ariboyarsky** [uchicago-computation-workshop/luis_bettencourt#11]
You mention in the paper that economic sorting is a highly diverse mechanism that would require varying model for different urban localities. Given this, do you think it is possible to use a similar technique to investigate  income segregation in the United States at-large. Furthermore, how does this play into studies of inequality that hope to understand income clustering among the wealthy in the United States. 

What exogenous factor would you point to that may drive this spatial distribution. How can this approach be used to better shed light on/identify these effects?

**Comment by lmbett** [comment]
Absolutely, you can consider the country, the state, the county, the city... the block. 

As to exogenous factor: they may not be entirely exogenous, but real estate markets (prices) drive segregation (and are driven by it). Similarly for quality of urban services (rating of schools) etc.
